modules/session-storage/pages/udf.adoc

```diff
@@ -0,0 +1,144 @@
+= Using User-Defined Functions with a Session Store
+:page-aliases: tutorials:session-storage-tutorial:udf
+
+== Overview
+
+This tutorial is part of the main Session Store tutorial (both link:http://docs.couchbase.com/tutorials/session-storage/aspnet.html[ASP.NET] and link:http://docs.couchbase.com/tutorials/session-storage/java.html[Java]). You should be able to follow along if you haven't gone through those tutorials yet, but reading those tutorials first will give you more context.
```

Comment:
```suggestion
This tutorial is part of the main Session Store tutorial (both xref:aspnet.adoc[ASP.NET] and xref:java.adoc[Java]). You should be able to follow along if you haven't gone through those tutorials yet, but reading those tutorials first will give you more context.
```

Reply:
Already fixed these in a different branch